crowbar_framework/app/models/service_object.rb

```diff
@@ -923,41 +923,7 @@ def self.proposal_to_role(proposal, bc_name)
     RoleObject.new role
   end
 
-  #
-  # After validation, this is where the role is applied to the system The old
-  # instance (if one exists) is compared with the new instance.  roles are
-  # removed and delete roles are added (if they exist) for nodes leaving roles
-  # roles are added for nodes joining roles.  Calls chef-client on nodes
-  #
-  # This function can be overriden to define a barclamp specific operation.  A
-  # call is provided that receives the role and all string names of the nodes
-  # before the chef-client call
-  #
-  # The in_queue signifies if apply_role was called from deployment queue's
-  # process_queue, and prevents recursion.
-  #
-  # The bootstrap parameter tells if we're in bootstrapping mode, in which case
-  # we simply do not run chef.
-  def apply_role(role, inst, in_queue, bootstrap = false)
-    Rails.logger.debug "apply_role(#{role.name}, #{inst}, #{in_queue}, #{bootstrap})"
-    Rails.logger.progress("Starting to apply role #{role.name}")
-
-    # Variables used in the global ensure
-    apply_locks = []
-    applying_nodes = []
-
-    # Cache some node attributes to avoid useless node reloads
-    node_attr_cache = {}
-
-    # experimental option
-    skip_unready_nodes_enabled = Rails.application.config.experimental.fetch(
-      "skip_unready_nodes", {}
-    ).fetch("enabled", false)
-
-    skip_unchanged_nodes_enabled = Rails.application.config.experimental.fetch(
-      "skip_unchanged_nodes", {}
-    ).fetch("enabled", false)
-
+  def looking_up_data_and_checks(role, old_role, inst, in_queue, bootstrap)
```

Comment:
I'd name this function in "verb" form (e.g. `lookup_data_and_run_checks`) to match the rest.

Reply:
done!